src/event/trigger.js

```diff
@@ -136,7 +139,17 @@ jQuery.extend( jQuery.event, {
 
 					// Prevent re-triggering of the same event, since we already bubbled it above
 					jQuery.event.triggered = type;
+
+					if ( event.isPropagationStopped() ) {
+						lastElement.addEventListener( type, stopPropagationCallback );
```

Comment:
@dmethvin I will try to add a test-case for the `lastElement === undefined` case. But I don't yes see how this could happen. How can the propagation of the incoming `even` already be stopped?